StackExchange.Redis/StackExchange/Redis/KeyspaceIsolation/DatabaseWrapper.cs

```diff
@@ -27,6 +27,33 @@ public RedisValue DebugObject(RedisKey key, CommandFlags flags = CommandFlags.No
             return Inner.DebugObject(ToInner(key), flags);
         }
 
+          public long GeoAdd(RedisKey key, RedisValue[] values, CommandFlags flags = CommandFlags.None)
+        {
+            return Inner.GeoAdd(ToInner(key), values, flags);
+        }
+
+        public RedisValue GeoDist(RedisKey key, RedisValue value0, RedisValue value1, GeoUnit geoUnit = GeoUnit.m,

```

Comment:
Naming: I've tended to use fully expanded names for clarity; `Geo` is fine, but this should probably be `GeoDistance`